packages/@webex/plugin-meetings/src/locus-info/selfUtils.ts

```diff
@@ -419,20 +419,27 @@ SelfUtils.mutedByOthersChanged = (oldSelf, changedSelf) => {
     throw new ParameterError('New self must be defined to determine if self was muted by others.');
   }
 
+  const isModifiedBySelf = changedSelf.selfIdentity === changedSelf.modifiedBy;
```

Comment:
I couldn't do the full check if we should ignore or not here, because selfUtils.ts doesn't have a reference to the meeting or MuteState class, so I've added the field `isModifiedBySelf` that's emitted with the `SELF_REMOTE_MUTE_STATUS_UPDATED` event and I've put all the logic to decide if we ignore the Locus DTO update or not in `MuteState.shouldIgnoreRemoteMuteUpdate()`